sdk/include/opentelemetry/sdk/trace/recordable.h

```diff
@@ -21,6 +23,24 @@ class Recordable
 public:
   virtual ~Recordable() = default;
 
+  /**
+   * Set a trace id for this span.
+   * @param trace_id the trace id to set
+   */
+  virtual void SetTraceId(opentelemetry::trace::TraceId trace_id) noexcept = 0;
+
+  /**
+   * Set a span id for this span.
+   * @param span_id the span id to set
+   */
+  virtual void SetSpanId(opentelemetry::trace::SpanId span_id) noexcept = 0;
+
+  /**
+   * Set a parent span id for this span.
+   * @param parent_span_id the parent span id to set
+   */
+  virtual void SetParentSpanId(opentelemetry::trace::SpanId parent_span_id) noexcept = 0;
```

Comment:
Could we combine SetTraceId, SetSpanId, SetParentSpanId into something like SetSpanContext(trace_id, span_id, parent_id)? For eager serialization implementations, it's common for them to be written together as a group.

Reply:
Makes sense. I think `SetSpanContext` is a bit ambiguous, as the concept of [SpanContext](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/api.md#spancontext) has a different meaning int the spec. I went with `SetIds` instead.